fish_right_prompt.fish

```diff
@@ -24,6 +23,15 @@ function fish_right_prompt
       end
     end
 
-    echo -n -s $ruby_info $normal
+    if [ "$theme_display_rbenv_with_gemfile_only" = 'yes' ]
+      if test -f Gemfile
+        echo -n -s $ruby_info $normal
+      end
+      return
```

Comment:
i think i didn't need to output anything if `Gemfile` didn't exist and `theme_display_rbenv_with_gemfile_only` set to `yes`, but anyway there is no different i will do it